src/Logging/test/ConsoleLoggerTest.cs

```diff
@@ -377,19 +378,65 @@ public void WriteCore_LogsCorrectTimestamp(ConsoleLoggerFormat format, LogLevel
             switch (format)
             {
                 case ConsoleLoggerFormat.Default:
-                {
-                    Assert.Equal(3, sink.Writes.Count);
-                    Assert.Matches("^\\d{14}\\s$", sink.Writes[0].Message);
-                    Assert.StartsWith(levelPrefix, sink.Writes[1].Message);
-                }
-                break;
+                    {
+                        Assert.Equal(3, sink.Writes.Count);
+                        Assert.StartsWith(levelPrefix, sink.Writes[1].Message);
+                        Assert.Matches("^\\d{4}\\D\\d{2}\\D\\d{2}\\D\\d{2}\\D\\d{2}\\D\\d{2}\\D\\d{2}\\s$", sink.Writes[0].Message);
+                        var parsedDateTime = DateTimeOffset.Parse(sink.Writes[0].Message.Trim());
+                        Assert.NotEqual(0, parsedDateTime.Offset.Hours);
+                    }
+                    break;
                 case ConsoleLoggerFormat.Systemd:
-                {
-                    Assert.Single(sink.Writes);
-                    Assert.Matches("^<\\d>\\d{14}\\s[^\\s]", sink.Writes[0].Message);
-                    Assert.StartsWith(levelPrefix, sink.Writes[0].Message);
-                }
-                break;
+                    {
+                        Assert.Single(sink.Writes);
+                        Assert.StartsWith(levelPrefix, sink.Writes[0].Message);
+                        var regexMatch = Regex.Match(sink.Writes[0].Message, "^<\\d>(\\d{4}\\D\\d{2}\\D\\d{2}\\D\\d{2}\\D\\d{2}\\D\\d{2}\\D\\d{2})\\s[^\\s]");
```

Comment:
Apply the similar `Assert.Matches` as above.

Reply:
I'd love to but then I cannot do `regexMatch.Groups` to retrieve the stamp in systemd logs.

Reply:
Ups...then it's fine.